components/organisms/NavBarWrapper.tsx

```diff
@@ -0,0 +1,28 @@
+"use client";
+
+import { useEffect, useState } from "react";
+import HamburgerMenu from "./HamburgerMenu";
+import { NavBar } from "./NavBar";
+
+export default function NavBarWrapper() {
+  const [isMobile, setIsMobile] = useState<boolean>(false);
+
+  useEffect(() => {
+    const handleResize = () => {
+      setIsMobile(window.innerWidth <= 600);
+    };
+
+    // Add event listener for window resize
+    window.addEventListener("resize", handleResize);
+
+    // Initial check for mobile screen on component mount
+    handleResize();
+
+    // Clean up the event listener on component unmount
+    return () => {
+      window.removeEventListener("resize", handleResize);
+    };
+  }, []);
+  // if is mobile return hamburger menu else return navbar
+  return <>{isMobile ? <HamburgerMenu /> : <NavBar />}</>;
```

Comment:
Two issues I see here:

1. This entirely removes the NavBar and its styling from the mobile design. I encourage you to come up with a solution here that only switches out the menu links for the menu button on small screen sizes. (Hint: display both the links and the menu button, and use Tailwind classes to enable one or the other based on screen size. Use CSS to handle the screen width, not JavaScript.)
2. We don't need the menu button yet since we don't have enough links to warrant it, so maybe let's get this to a shippable state and then comment it out until we need it?